pkg/repositories/transformers/workflow_test.go

```diff
@@ -72,8 +72,20 @@ func TestFromWorkflowModel(t *testing.T) {
 		Name:         "name",
 		Version:      "version",
 	}, workflow.Id))
+
+	var workflowInterface core.TypedInterface
```

Comment:
no need to unmarshal for this test, you can always re-use `testutils.GetWorkflowRequest().Spec.Template.Interface`
from https://github.com/flyteorg/flyteadmin/blob/dc8cc9d32b30e82c7d147a1ac88377a41165b195/pkg/manager/impl/testutils/mock_requests.go#L98-L113

Reply:
Makes sense, will fix